libp2p/transports/wstransport.nim

```diff
@@ -132,6 +134,18 @@ method start*(
   await procCall Transport(self).start(addrs)
   trace "Starting WS transport"
 
+  # use autotls if possible (and if cert/key not specified)
+  if not self.secure and not self.autotls.isNil():
+    try:
+      self.tlsCertificate = await self.autotls.getCertWhenReady()
+      self.tlsPrivateKey = self.autotls.getTLSPrivkey()
```

Comment:
so ws transport sets `self.tlsCertificate` and `self.tlsPrivateKey`  properties
and how are these used?